apps/api/src/app/integrations/e2e/get-active-integration.e2e.ts

```diff
@@ -0,0 +1,87 @@
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+import { ChannelTypeEnum, EmailProviderIdEnum } from '@novu/shared';
+import { IntegrationRepository } from '@novu/dal';
+
+describe('Get Active Integrations - /integrations/active (GET)', function () {
+  let session: UserSession;
+  const integrationRepository = new IntegrationRepository();
+
+  beforeEach(async () => {
+    session = new UserSession();
+    await session.initialize();
+  });
+
+  it('should get active integrations', async function () {
+    const activeIntegrations = (await session.testAgent.get(`/v1/integrations/active`)).body.data;
+
+    const { inAppIntegration, emailIntegration, smsIntegration, chatIntegration, pushIntegration } =
+      splitByChannels(activeIntegrations);
+
+    expect(inAppIntegration.length).to.equal(1);
+    expect(emailIntegration.length).to.equal(1);
+    expect(smsIntegration.length).to.equal(1);
+    expect(pushIntegration.length).to.equal(1);
+    expect(chatIntegration.length).to.equal(2);
+
+    expect(inAppIntegration[0].selected).to.equal(true);
+    expect(emailIntegration[0].selected).to.equal(true);
+    expect(smsIntegration[0].selected).to.equal(true);
+    expect(pushIntegration[0].selected).to.equal(true);
+
+    const oneSelected = chatIntegration.some((integration) => integration.selected === true);
+    const oneNotSelected = chatIntegration.some((integration) => integration.selected === false);
```

Comment:
Should we just count the ocurrences of the condition checked so in the future if we improve the test for multiple providers with some selected and some not it will be easier to update the test?

Reply:
I was not aware that in the future we will support 'some selected' i thought there will be only one selected, in order to add such tests I will need more information regarding the future implementation, regardless I think this test will be enough for this stage.